erpc_c/port/erpc_config_internal.h

```diff
@@ -195,6 +195,25 @@
     #define ERPC_PRE_POST_ACTION_DEFAULT (ERPC_PRE_POST_ACTION_DEFAULT_DISABLED)
 #endif
 
+#if !defined(erpc_assert)
+    #if ERPC_HAS_FREERTOSCONFIG_H 
+        #ifdef __cplusplus
+            extern "C" {
+        #endif
+        #include "FreeRTOSConfig.h"
+        #ifdef __cplusplus
+            }
+        #endif
+        #define erpc_assert(condition) configASSERT(condition)
+    #elif defined(ERPC_THREADS) && (ERPC_THREADS == ERPC_THREADS_MBED)
+        #include "platform/mbed_assert.h"
+        #define erpc_assert(condition) MBED_ASSERT(condition)
+    #else
+        #include <cassert>
```

Comment:
#include <cassert> does not work in c projects (for instance when rpmsg_env_bm.c is involved in erpc example project the cassert C++ include can't be found, causing build error. When replaced by #include "assert.h" all works fine. 
@Hadatko is it ok to replace #include <cassert> by #include "assert.h"? Could this cause any other issue?

Reply:
> #include cassert does not work in c projects (for instance when rpmsg_env_bm.c is involved in erpc example project the cassert C++ include can't be found, causing build error. When replaced by #include "assert.h" all works fine. @Hadatko is it ok to replace #include cassert by #include "assert.h"? Could this cause any other issue?

My new commit should fixed this. Do you agree?

Reply:
Yes, seems to be a good solution.